### PR TITLE
Conformance results for v1.10/cdk

### PR DESCRIPTION
**Pull request by ktsakalozos**
Conformance results for Canonical Distribution of Kubernetes - Kubernetes version: v1.10

### CODE REVIEW

File: v1.10/cdk/PRODUCT.yaml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+vendor: Canonical Ltd
+name: Canonical Distribution of Kubernetes
+version: v1.10.0
+website_url: https://www.ubuntu.com/kubernetes
+documentation_url: https://kubernetes.io/docs/getting-started-guides/ubuntu/
+product_logo_url: https://github.com/kubernetes/kubernetes/blob/master/logo/logo.svg
```
Comment (dankohn) [inline v1.10/cdk/PRODUCT.yaml]: Please update the logo, as that belongs to Kubernetes, not the Canonical Distribution of Kubernetes.

We are currently using:
https://landscape.cncf.io/logos/canonical-distribution-of-kubernetes.svg
https://landscape.cncf.io/grouping=organization&organization=canonical&selected=canonical-distribution-of-kubernetes

We can continue to do so, if you want.

Comment (ktsakalozos) [reply v1.10/cdk/PRODUCT.yaml]: Thank you for spotting this out. Just updated the logo as requested.

Comment (ktsakalozos) [reply v1.10/cdk/PRODUCT.yaml]: Although the info displayed in https://landscape.cncf.io/grouping=organization&organization=canonical&selected=canonical-distribution-of-kubernetes are not right. Canonical Headcount is 500+ and the linkedin link is broken.

Comment (dankohn) [reply v1.10/cdk/PRODUCT.yaml]: @ktsakalozos The landscape pulls the Canonical info from: https://www.crunchbase.com/organization/canonical-ltd

I just updated the LinkedIn link. If you want to tell me the correct headcount, I can adjust that too, or you can do it yourself. We'll then fetch the corrected info tonight.

Comment (ktsakalozos) [reply v1.10/cdk/PRODUCT.yaml]: Thank you, we will update the canonical ourselves.

**Comment by taylorwaggoner** [comment]
You are now Certified Kubernetes

**Comment by ktsakalozos** [comment]
Thank you!

**Pull request merged**